extensions/ql-vscode/src/vscode-tests/no-workspace/activation/activation.test.ts

```diff
@@ -1,13 +1,15 @@
-import * as assert from "assert";
+// This file needs to be located in a separate directory. This will ensure that this
```

Comment:
Sorry if I'm being dumb, but this file needs to be in a separate directory from what?

Reply:
Sorry that it's unclear, I've improved the comment. It should be in a separate directory from any other tests. `jest-runner-vscode` will start a new VSCode instance for every directory containing tests, so by having this in a separate directory it ensures the extension hasn't been activated by another test file yet.